lib/DDG/Spice/Cocoapods.pm

```diff
@@ -0,0 +1,38 @@
+package DDG::Spice::Cocoapods;
+# ABSTRACT: Write an abstract here
```

Comment:
@soleo you can remove these comments, they're just for you to see/read when developing your IA.